backend/substrapp/tasks/tasks.py

```diff
@@ -227,25 +232,30 @@ def fetch_model(channel_name, parent_tuple_type, authorized_types, input_model,
 def fetch_models(channel_name, tuple_type, authorized_types, input_models, directory):
 
     models = []
+    exceptions = []
 
     for input_model in input_models:
-        proc = ExceptionThread(target=fetch_model,
-                               args=(channel_name, tuple_type, authorized_types, input_model, directory))
-        models.append(proc)
+        # Authentification should be retrieved before sending the function fetch model to a subProcess
+        # as, fetching it from django models give random results
+        # Don't know why
```

Comment:
Just curious @Kelvin-M what kind of random results you get here? I think even we can merge this we should do further investigations in orderer to understand what is happening

Reply:
Retrieving sometime a wrong password for a particular `node_id` instead of the good one :(